packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts

```diff
@@ -38,6 +38,7 @@ export type InstructionDisplayText = {
   instructionListStepTwoText?: string;
   instructionListStepThreeText?: string;
   instructionListStepFourText?: string;
+  legalText?: string;
```

Comment:
Doesn't placing this here mean that customers can override it? Should they be able to?

Reply:
Yes, they need to be able to translate and confirmed with legal that customers have the ability to override.